sdk/cosmos/azure-cosmos/test/test_full_text_policy.py

```diff
@@ -0,0 +1,264 @@
+                {"path": "/abstract"}
+            ]
+        }
+        created_container = self.test_db.create_container(
+            id='full_text_container' + str(uuid.uuid4()),
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy,
+            indexing_policy=indexing_policy
+        )
+        properties = created_container.read()
+        assert properties["fullTextPolicy"] == full_text_policy
+        assert properties["indexingPolicy"]['fullTextIndexes'] == indexing_policy['fullTextIndexes']
+
+        # Replace the container with new policies
+        full_text_policy['fullTextPaths'][0]['path'] = "/new_path"
+        indexing_policy['fullTextIndexes'][0]['path'] = "/new_path"
+        replaced_container = self.test_db.create_container(
+            id=created_container.id,
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy,
+            indexing_policy=indexing_policy
+        )
+        properties = replaced_container.read()
+        assert properties["fullTextPolicy"] == full_text_policy
+        assert properties["indexingPolicy"]['fullTextIndexes'] == indexing_policy['fullTextIndexes']
+        self.test_db.delete_container(created_container.id)
+
+    def test_fail_create_full_text_policy(self):
+        # Pass a full text policy with a wrongly formatted path
+        full_text_policy_wrong_path = {
+            "defaultLanguage": "en-US",
+            "fullTextPaths": [
+                {
+                    "path": "abstract",
+                    "language": "en-US"
+                }
+            ]
+        }
+        try:
+            self.test_db.create_container(
+                id='full_text_container',
+                partition_key=PartitionKey(path="/id"),
+                full_text_policy=full_text_policy_wrong_path
+            )
+            pytest.fail("Container creation should have failed for lack of embedding policy.")
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 400
+            assert "The Full Text Policy contains an invalid Path: abstract" in e.http_error_message
+
+        # Pass a full text policy without language attached to the path
+        full_text_policy_no_langs = {
+            "defaultLanguage": "en-US",
+            "fullTextPaths": [
+                {
+                    "path": "/abstract"
+                }
+            ]
+        }
+        try:
+            self.test_db.create_container(
+                id='full_text_container',
+                partition_key=PartitionKey(path="/id"),
+                full_text_policy=full_text_policy_no_langs
+            )
+            pytest.fail("Container creation should have failed for lack of embedding policy.")
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 400
+            assert "The Full Text Policy contains invalid syntax" in e.http_error_message
```

Comment:
Where these error messages constructed? Can we update the error message with more details? Users many do not know exactly what was missing.

Reply:
it's all in the service (backend) - we have already provided this feedback to them, but yep completely agreed. this is just what they are for now